src/inverseRobot.js

```diff
@@ -24,7 +24,23 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+
+  for (const value of robotValues) {
+    if (robotValues.indexOf(value) !== robotValues.lastIndexOf(value)) {
```

Comment:
use `for in` loop and remove robotValues variable
also, use `hasOwnProperty` instead of `indexOf` and `lastIndexOf`